--- a/setup.py
+++ b/setup.py
@@ -25,6 +25,11 @@
 def read(fname):
     return open(os.path.join(os.path.dirname(__file__), fname)).read()
 
+dst = 'debian/openstack-dashboard/var/lib/dash'
+os.system('rm -rf %s' % dst)
+shutil.copytree('dashboard', '%s/dashboard' % dst)
+shutil.copytree('local', '%s/local' % dst)
+shutil.copyfile('manage.py', '%s/dashboard/manage.py' % dst)
 
 setuptools.setup(
     name = 'openstack-dashboard',
